cli/io/cbt.py

```diff
@@ -0,0 +1,189 @@
+                data = {
+                    "job_id": ctx.get("config", {}).get("job_id", None),
+                    "started_at": ctx.get("config", {}).get("timestamp", None),
+                    "benchmark_mode": cbt_results.get("Benchmark_mode", None),
+                    "seq": cbt_results.get("seq", None),
+                    "total_cpu_cycles": cbt_results.get("total_cpu_cycles", None),
+                    "branch": ctx.get("config", {}).get("branch", None),
+                    "sha1": ctx.get("config", {}).get("sha1", None),
+                    "os_type": ctx.get("config", {}).get("os_type", None),
+                    "os_version": ctx.get("config", {}).get("os_version", None),
+                    "machine_type": ctx.get("config", {}).get("machine_type", None),
+                    "benchmark": benchmark["benchmarks"],
+                    "results": cbt_results.get("results", None),
+                }
+                response = requests.post(
+                    self.endpoint_url, json=data, headers=self.headers, auth=self.auth
+                )
+                import pdb
+
+                pdb.set_trace()
+
+                if response.status_code == 201:
+                    log.info("Data inserted successfully.")
+                else:
+                    log.info(f"Error inserting data: {response}")
+
+    def read_results(self, json_output_paths):
+        """
+        json_output_paths = "/tmp/test_cbt_3/results/00000000/id-01984f6e/"
+                            "json_output.0.ceph-amk-cbt-c9dbme-node1-installer"
+        """
+        results = []
+        for json_output_path in json_output_paths:
+            if json_output_path:
+                path_full = Path(json_output_path)
+                match = re.search(r"/json_output\.(?P<json>\d+)", json_output_path)
+                if match:
+                    Benchmark_mode = (
+                        path_full.parent.name
+                        if path_full.parent.name in ["rand", "write", "seq"]
+                        else "fio"
+                    )
+                    seq = match.group("json")
+
+                    results.append(
+                        {
+                            "results": self.read_json_file(json_output_path),
+                            "Benchmark_mode": Benchmark_mode,
+                            "seq": seq,
+                        }
+                    )
+        return results
+
+    def enable_epel(self):
+        EPEL_REPOS = {
+            "7": "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm",
+            "8": "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm",
+            "9": "https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm",
+        }
+
+        # Epel Repo
+        self.execute(
+            sudo=True,
+            cmd=f"dnf install {EPEL_REPOS[self.primary_client.distro_info['VERSION_ID'][0]]} -y",
+            check_ec=False,
+        )
```

Comment:
I believe this code can exists outside and it would be helpful for other comments.